docs/privacy-and-security.md

```diff
@@ -73,6 +73,37 @@ Some editors keep usage history stored on disk for future use. This can be a
 security risk in the sense that sensitive information can leak via recent
 search patterns or editor commands.
 
+### Visual Studio Code
+
+Visual Studio Code has a built-in local file history. When enabled, the file
+contents of an editor that is saved will be stored to a history location to be
+able to restore or review the contents later.
+
+This leaky feature can be disabled for all files (including those for which
+you might need the history) by unchecking the `workbench.localHistory.enabled`
+setting in the [Settings editor](https://code.visualstudio.com/docs/getstarted/settings#_settings-editor).
+
+Alternatively, you can configure a [pattern](https://code.visualstudio.com/docs/editor/codebasics#_advanced-search-options) in the `workbench.localHistory.exclude`
+setting for excluding only specific files from local file history. For excluding
+unencrypted temporary file generated by `jrnl`, you can set the `**/jrnl*.jrnl`
+pattern for the `workbench.localHistory.exclude` setting in the [Settings editor](https://code.visualstudio.com/docs/getstarted/settings#_settings-editor).
+
+!!! note
+    On Windows, the history location is typically found at
+    `%APPDATA%\Code\User\History`.
```

Comment:
Actually, I found this position by manually searching in the local data folder of VS Code. There is no reference in the documentation as to where the location actually is.
Now I have just found a note about this in the [release notes of version 1.66](https://code.visualstudio.com/updates/v1_66#_local-history) (the one in which local history was introduced):

> Local history entries are stored in different locations depending on your use of VS Code. When opening local files, the entries are persisted in the local user data folder and when opening remote files, they will be stored on the remote user data folder. When no file system is available (for example, in certain cases when using VS Code for Web), entries are stored into IndexedDB.

Reply:
Thanks for the info, and for testing this. That works for me. That's interesting about the remote data and no-file-system scenarios, but I think it's fine not to mention since the jrnl temp files are local.